scipy/optimize/_lsq/bvls.py

```diff
@@ -14,17 +14,17 @@ def compute_kkt_optimality(g, on_bound):
     return np.max(g_kkt)
 
 
-def bvls(A, b, x_lsq, lb, ub, tol, max_iter, verbose):
+def bvls(A, b, x_lsq, lb, ub, tol, max_iter, verbose, rcond=None):
```

Comment:
Is `rcond` parameter significant for this algorithm? If so we should pass it through `lsq_linear`.

Can we get away with default `rcond=-1` which uses machine epsilon to cutoff small singular values? I wouldn't introduce another parameter if there are no cases for it.